crates/starknet-server/tests/test_estimate_message_fee.rs

```diff
@@ -104,6 +104,8 @@ mod test_estimate_message_fee {
             .await
             .expect_err("Error expected");
 
+        println!("{:?}", err);
```

Comment:
I'm also interested in this println :D

Reply:
https://github.com/0xSpaceShard/starknet-devnet-rs/commit/a142178490aa52cc304630bcfe80109ad50e89b8